dask/array/__init__.py

```diff
@@ -29,7 +29,7 @@
             isreal, iscomplex, isfinite, isinf, isneginf, isposinf, isnan, signbit,
             copysign, nextafter, spacing, ldexp, fmod, floor, ceil, trunc, degrees,
             radians, rint, fix, angle, real, imag, clip, fabs, sign, absolute,
-            i0, sinc, nan_to_num, frexp, modf, divide, frompyfunc, float_power)
+            i0, sinc, nan_to_num, frexp, modf, divide, frompyfunc, float_power, divmod)
```

Comment:
You should also add divmod to the `docs/source/array-api.py` file in the list "Top level user functions"